blog/_posts/2017-09-25-serv0101-conference-blog.md

```diff
@@ -0,0 +1,67 @@
+---
+title: JSFoo 2017 - Team Serv0101 
+layouts: post
+created_at: Mon Sep 25 2017
+permalink: blog/2017-09-25-Team-serv0101
+current: blog
+author: Neha & Rakhi
+categories:
+- student-posts
+---
+
+## Team Serv0101 at JSFoo 2017
+<img src="/img/blog/2017/2017-09-25-Serv0101-JSfoo2017.png">
+<div class="image-credits">Screenshot</div>
+JSFoo is launched in 2011 which is a India’s first JavaScript conference. The JS community in India has grown phenomenally since then. JavaScript now pervades every aspect of web development – browsers, apps, front-end, backend, mobile and IoT, and there’s always scope to understand new ideas and solutions. The conference explores new ideas, implementing innovative solutions, and learning from experiences, especially negative ones!
+
+JSFoo was two-day conference(15th Sep 2017 - 16th Sep 2017), Bangalore, India.
+This year theme was building reliable web apps. Talks happened on these topics and more:
+* Tooling and best practices for measuring and monitoring on the web
+* Modern programming languages like ES6/7 and TypeScript
+* Architecture and the practice of programming
+* Specific libraries and frameworks
+* Browser APIs like WebRTC, ServiceWorkers, Web Components, and Push API
+* Accessibility and localization
+* Game programming
+* Visualisation and animation on the web using JavaScript
+
+#### Every talk and workshop we attended there was awesome but we found following two very interesting.
+**1. Tiny Computers, JavaScript and MIDI by <a href="https://twitter.com/georgemandis">George Mandis</a>**
+<img src="/img/blog/2017/2017-09-25-Serv0101-George.jpeg">
+<div class="image-credits">Photo taken by Neha</div>
+George Mandis is a main developer of Konami.js. Being a konami js lover it was really nice to see his talk and meet him in person.
+In his JSFoo talk, we explored why MIDI implementation and prevalence in hardware makes uniquely intriguing and accessible platform for creative coding.
+
+Tiny computers such as the RaspberryPi, Arduino and C.H.I.P. have finally reached a level of performance and affordability that makes hardware experimentation accessible to everyone. At the same time, JavaScript has emerged as the lingua franca of web programming and can be found in many places beyond just the browser.
+
+* MIDI — a niche protocol that’s been around since 1983 and was originally designed for musical instruments to communicate with one another.
+
+* Tiny Computers + JavaScript + MIDI - create great platform for both education and entertainment purposes. 
+
+**2. Building Immersive Virtual Reality experiences quickly with WebVR by <a href="https://twitter.com/ram_gurumukhi">Ram Vaishnav</a>**
+
+Ram Vaishnav says - "I believe that Virtual Reality is going to become a primary platform soon, which will completely change the way we work, play and communicate digitally."
+
+* What is WebVR?
```

Comment:
Bullet points looks odd when we have title and context both. Can you please use mark down style headers for these two?